decentraland/unity-explorer
# fix: added reflection probe

## What does this PR change?

- Modified Skybox Shader to use fog color as floor color
- Added reflection probe to the character object
- Reflection Probe script forces environment reflection to custom and updates the custom reflection texture every x amount of seconds.
- Its set to 0.1 seconds refresh and should not be affecting performance
- Now reflective materials should react to skybox changes

- Copied Ocean material to customize colors and features
- Extended the size of the world and city oceans

## How to test the changes?

1. Launch the explorer
2. Go to the edge of the world and see the sea
3. Change the skybox time in the debug panel and check reflections colorize with the skybox colors
4. Go to DCL expererience and check reflective materials (like the water of the top of the clouds, or the metallics)
5. Those should also react to time change

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
